kafka-ui-react-app/src/components/common/Button/Button.tsx

```diff
@@ -20,5 +22,19 @@ export const Button: React.FC<Props> = ({ to, ...props }) => {
       </Link>
     );
   }
-  return <StyledButton type="button" {...props} />;
+  return (
```

Comment:
i think we should disable the button when it is in progress,
`<StyledButton type="button" {...props} disabled={props.disabled || props.inProgress}`

Reply:
Updated + added test for this behavior